clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultEventHandler.java

```diff
@@ -141,11 +77,25 @@ public <T> T addAndGet(final CompletableApplicationEvent<T> event, final Timer t
         return event.get(timer);
     }
 
-    public void close() {
-        try {
-            backgroundThread.close();
-        } catch (final Exception e) {
-            throw new RuntimeException(e);
-        }
+    public void close(final Duration timeout) {
+        Objects.requireNonNull(timeout, "Duration provided to close must be non-null");
+
+        closer.close(
+                () ->  {
+                    log.info("Closing the default consumer event handler");
```

Comment:
debug level better? (same for all other log lines in this func)

Reply:
current log is at trace level: `log.trace("Closing the Kafka consumer");`

Reply:
I'll change the log line at the start of the method from `info` to `trace` and leave the log line at the end of the method as `debug`. In so doing, both will match the levels in `KafkaConsumer.close()`.